Purchasing.Mvc/Services/IetWsDirectorySearchService.cs

```diff
@@ -67,16 +67,20 @@ private async Task<Person> GetByEmail(string email)
                 throw new Exception("More than 1 unique kerb values found.");
             }
             var ucdKerbPerson = ucdKerbResult.ResponseData.Results.First();
+            return GetPersonDetails(ucdKerbPerson, ucdContact);
+        }
+
+        private static Person GetPersonDetails(KerberosResult ucdKerbPerson, ContactResult ucdContact)
```

Comment:
only needs to be static if you are calling from a static context.  In this case you are not so you shouldn't need/have it.